syntax = "proto3";

package exporter;

service PdfExporter {
  rpc ExportPdf (ExportRequest) returns (ExportResponse);
}

message ExportRequest {
  string md_path = 1;
  string config_path = 2;
}

message ExportResponse {
  string pdf_path = 1;
}

